### ISSUE TITLE
Selecting notifications to send

### ISSUE THREAD

**Issue by asupreme1337** [louislam/uptime-kuma#1582]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

Other

### 🔖 Feature description

In the column "**your message prefix**" when setting up notifications via **Discord**, I would like to see the setting of different types of alerts (service is available / service is unavailable) so that the user is pinged (marked) only if the service has become unavailable. Because, as I understand it, now this function works like this: for any notification sent to Discord (up/down), the personal prefix of the message will be used. It's a little superfluous.

### ✔️ Solution

Approximate setting of the personal prefix of the Discord message:
• **Message if the service has become unavailable**: "_@example#1337 - pay attention_"
• **A message if the service is available again**: ....

### 📝 Additional Context

I hope I have presented my idea correctly. If something is unclear - write about it, I will try to reformulate my thought.

**Comment by MadGodNerevar** [comment]
I agree with this, would be nice to receive a different message based on the status.

the way I would personally like to work it is:

Status = DOWN
please check status of “blah” network VM all that

status = UP
we are now back to normal 

something like this would be great 😃 
Uptime Kuma is great and the notification support is truly something
